w3af/core/controllers/core_helpers/status.py

```diff
@@ -166,13 +166,13 @@ def get_rpm(self):
                  start of the scan.
         """
         if self._start_time_epoch is None:
-            raise RuntimeError('Can NOT call get_run_time before start().')
-        
+            return None
```

Comment:
Hmmm, why this change? I believe that the exception was correct, but if you've got a good reason we can keep it.